subt/main.py

```diff
@@ -259,6 +269,16 @@ def follow_wall(self, radius, right_wall=False, timeout=timedelta(hours=3), dist
                     print(self.time, "VIRTUAL BUMPER - collision")
                     self.go_straight(-0.3, timeout=timedelta(seconds=10))
                     break
+
+                if self.lora_cmd is not None:
+                    # the "GoHome" command must be accepted only on the way there and not on the return home
```

Comment:
This comment is correct. How do we make it a reality?

Reply:
Oh, `dist_limit` is None on the way there. I see. It is not the prettiest thing in the world, but it will make do for tomorrow.